cog/slots.py

```diff
@@ -128,10 +138,15 @@ async def _roll_reels(self, reels, msg, ctx):
             else:
                 reel2_border = ":"
 
-            await asyncio.sleep(1)
+            await asyncio.sleep(0.2)
```

Comment:
This sleep function should be at the start of the while loop (or so). What happens is that the slots are assigned and sent to chat, but since you don't asyncio.sleep(), it immediately modifies the message that was sent again. So instead of the delay of ticks being like 1, 1, 1 for 3 ticks, it's more like 0, 1, 1.